baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/JUnit5RuleUsage.java

```diff
@@ -62,9 +62,9 @@ public Description matchClass(ClassTree tree, VisitorState state) {
     }
 
     static Matcher<ClassTree> hasVariable(Matcher<VariableTree> matcher) {
-        return (classTree, state) -> classTree.getMembers().stream()
-                .filter(tree -> tree instanceof VariableTree)
-                .anyMatch(tree -> matcher.matches((VariableTree) tree, state));
+        return (classTree, state) ->
+                classTree.getMembers().stream().filter(tree -> tree instanceof VariableTree).anyMatch(tree ->
+                        matcher.matches((VariableTree) tree, state));
```

Comment:
@dansanduleac what do you think of this one? dots were aligned and no longer are 😭

Reply:
yeah... I think it's because recently we've allowed it to break at lambda bodies.
I'm surprised that it allowed the dot in `.anyMatch` to be inlined so far in, but i think this is because we only applied the column limit to the logic that tries to one-line the whole level, whereas this code path is different.